cpp/src/arrow/parquet/parquet-io-test.cc

```diff
@@ -150,10 +189,28 @@ TEST_F(TestParquetIO, SingleColumnInt64ChunkedRead) {
   }
 }
 
+TEST_F(TestParquetIO, SingleColumnInt64ChunkedTableRead) {
+  std::vector<int64_t> values(100, 128);
+  std::unique_ptr<ParquetFileReader> file_reader = Int64File(values, 4);
+
+  std::shared_ptr<Table> out;
+  ReadTableFromFile(std::move(file_reader), &out);
+  ASSERT_EQ(1, out->num_columns());
+  ASSERT_EQ(100, out->num_rows());
+
+  std::shared_ptr<ChunkedArray> chunked_array = out->column(0)->data();
+  ASSERT_EQ(1, chunked_array->num_chunks());
+  Int64Array* out_array = static_cast<Int64Array*>(chunked_array->chunk(0).get());
+  for (size_t i = 0; i < values.size(); i++) {
+    EXPECT_EQ(values[i], out_array->raw_data()[i]);
+  }
```

Comment:
Can you factor out this comparison into a helper function (as this is the same as above)?